modules/developer-cli-odo-creating-a-database-with-odo.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+
+[id="creating-a-database-with-odo_{context}"]
+
+= Creating a database with `odo`
+
+To create a database, you must have an access to the database Operator. For this example *Dev4Devs PostgreSQL Operator* is used.
+
+.Procedure
+
+
+. See the list of the services in your project:
+
+
+. Store the YAML of the service in a file:
++
+[source,terminal]
+----
+$ odo service create postgresql-operator.v0.1.1/Database --dry-run > db.yaml
+----
+
+. Add following values under `metadata:` section in the `db.yaml` file:
++
+[source,yaml]
+----
+  name: sampledatabase
+  annotations:
+    service.binding/db.name: 'path={.spec.databaseName}'
+    service.binding/db.password: 'path={.spec.databasePassword}'
+    service.binding/db.user: 'path={.spec.databaseUser}'
+----
++
+This configuration ensures that when a database service is started, appropriate annotations are added to it. Annotations help the Service Binding Operator in injecting values for `databaseName`, `databasePassword` and `databaseUser` into the application. 
```

Comment:
A small suggestion:

...Service Binding Operator in injecting the values for `databaseName`, `databasePassword`...

Reply:
LGTM